MsCorePkg/UnitTests/MsCorePkgHostTest.dsc

```diff
@@ -0,0 +1,48 @@
+## @file
+# Host Test DSC for the MsCore Package
+#
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+################################################################################
+[Defines]
+  PLATFORM_NAME                  = MsCorePkg
+  PLATFORM_GUID                  = 55442A6C-2455-414D-9842-23002F5B20C7
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010005
+  OUTPUT_DIRECTORY               = Build/MsCorePkg
```

Comment:
I don't think we have a firm convention here, but I think it would be nice to separate the builds from this DSC.

```suggestion
  OUTPUT_DIRECTORY               = Build/MsCorePkg/HostTest
```